src/Heap.py

```diff
@@ -0,0 +1,56 @@
+#  _*_coding:utf-8 _*_
+"""Create a heap data structure."""
+from __future__ import division
+import math
+
+
+class Heap(object):
+    """Create a heap data structure."""
+
+    def __init__(self):
+        """Init method for Heap class."""
+        self.hl = []
```

Comment:
From the assignment description: "The constructor for your heap should default to creating an empty heap, but allow for creating a populated given an iterable as an input.". This constructor does not accept any arguments.